test/Counter.test.ts

```diff
@@ -66,4 +88,90 @@ describe('Counter', function () {
 			expectResultValue(unsealedResult, 5n)
 		})
 	})
+
+	describe('Functionality (localcofhe)', function () {
+		beforeEach(async function() {
+			if (!isLocalCofhe()) {
```

Comment:
I would prefer to have a `localCofheOnly()` and a `cofheMocksOnly()` instead of this pattern. It doesn't allow us to add additional environments.

Maybe even something like `permittedCofheEnvironment("MOCK")`, we can discuss